goguen/formal-spec/transactions.tex

```diff
@@ -90,6 +90,8 @@ \section{Transactions}
   option will additionally allow users to write their own programmatic solutions
   to choosing for-fees inputs.
 
+  \item $\TxId$ is always computed from values of type $\Tx$, never from $\ShelleyTx$ or $\GoguenTx$.
```

Comment:
I don't think it's the best place for this, since the explanation is about `txid` the function (which always hashes the original serialization).

Reply:
I'm not sure where else to put it though. In the vicinity of `txid`, it's obvious from the type of `txid`.

Reply:
Move it to 4.1, before the note after the bullet points (before "Note that when submitting a transaction, ..")